modules/virt-about-eus-updates.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * virt/upgrading-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-eus-updates_{context}"]
+= About EUS-to-EUS updates
+
+Every other minor version of {product-title} is an Extended Update Support (EUS) version, but because Kubernetes design mandates serial minor version updates, you cannot directly update from one EUS version to the next. You must first update to the next minor version before you can proceed to the target EUS version.
```

Comment:
This is a bit misleading. It is giving me the impression that even 4.11->4.13 would be considered EUS. Which would not be correct.

Reply:
Great feedback, thank you!